lib/ansible/module_utils/common/respawn.py

```diff
@@ -66,6 +69,24 @@ def probe_interpreters_for_module(interpreter_paths, module_name):
         'PYTHONPATH': f'{_ANSIBLE_PARENT_PATH}:{PYTHONPATH}'.rstrip(': ')
     })
 
+    import_string = ''
+    if module_names is not None:
+        module_names.append('ansible.module_utils.basic')
+        import_string = f"import {', '.join(module_names)}"
+
+    if module_name is not None:
+        warnings.deprecate(
+            msg='Use of module_name is deprecated.',
+            version='2.24',
```

Comment:
Shouldn't this be 2.25, since this will end up in ansible-core 2.21?
```suggestion
            version='2.25',
```